### PR TITLE
Migrate `extjs_deftjs` example to `todomvc-app-css`

### PR DESCRIPTION
**Pull request by arthurvr**
Same thing as previous PRs, but now the `extjs_deftjs` example.

* Included `todomvc-app-css`, updated the version of `todomvc-common`. This includes the new UI :tada:.
* Replaced bower by NPM.
* Include all files that aren't needed in the repo in the gitignore.
* #1110

### CODE REVIEW

File: examples/extjs_deftjs/.gitignore [hunk 1]
```diff
@@ -4,3 +4,10 @@ coffee
 tools
 css/base.css
 css/bg.png
+
```
Comment (sindresorhus) [inline examples/extjs_deftjs/.gitignore]: the two last ones above can be removed

Comment (arthurvr) [inline examples/extjs_deftjs/.gitignore]: I'll do it, but if fact that has nothing to do with this PR right?

Comment (arthurvr) [inline examples/extjs_deftjs/.gitignore]: done :)

Comment (sindresorhus) [inline examples/extjs_deftjs/.gitignore]: Ah, you're right, it's a leftover from a previous change. But still nice to remove it though.

**Comment by arthurvr** [comment]
@sindresorhus Can you tick this one and `aria templates` in the list on #1110?

**Comment by sindresorhus** [comment]
Yes, sorry forgot. Done now :)

**Comment by arthurvr** [comment]
thanxx Sindre!

**Pull request merged**